MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -2580,6 +2539,52 @@ public void handleDayEnding(DayEndingEvent evt) {
         }
     }
 
+    /**
```

Comment:
Moved these functions into their own discrete methods, as it made sense to isolate them in case we ever want to move them into their own class (like the other nag dialogs).

Reply:
Just an FYI, not requesting any changes:

Private members of a class are only visible to instances of that class, so will not be available for unit testing.  
It is possible to write less-protected wrapper functions that then call the `private` methods just for testing purposes - that is, they can then be called in unit tests, but the private methods themselves will be invisible to unit testing code.

For that reason, it is common to see code that has unit tests get switched from `private` to `protected` - the Junit unit test code needs at least this much access to call the methods.
Package-private is another option _iff_ the unit tests can be placed in the package hierarchy of the  code under test, but I don't think we have that option with Junit 5.